third_party/2and3/redis/client.pyi

```diff
@@ -170,40 +170,51 @@ class Redis(object):
     def blpop(self, keys: Union[_Value, Iterable[_Value]], timeout: int = ...) -> Optional[Tuple[bytes, bytes]]: ...
     def brpop(self, keys: Union[_Value, Iterable[_Value]], timeout: int = ...) -> Optional[Tuple[bytes, bytes]]: ...
     def brpoplpush(self, src, dst, timeout=...): ...
-    def lindex(self, name, index): ...
-    def linsert(self, name, where, refvalue, value): ...
-    def llen(self, name): ...
+    def lindex(self, name: _Key, index: int) -> Optional[bytes]: ...
+    def linsert(self, name: _Key, where: Text, refvalue: _Value, value: _Value) -> int: ...
```

Comment:
Since `where` can only be `BEFORE` or `AFTER` (case insensitive), I suggest to use `Literal["before", "after", "BEFORE", "AFTER"]` as type. This also needs an `from typing_extensions import Literal` import.

Reply:
[`typing.Literal`](https://docs.python.org/3/library/typing.html#typing.Literal) was introduced only in Python 3.8, which means that it wouldn't work on 3.7 or 3.6 which are still supported. Do you still recommend using `typing.Literal` here?